apache-kafka-2/src/main/java/com/baeldung/kafka/message/ordering/serialization/JacksonDeserializer.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.kafka.message.ordering.serialization;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.kafka.common.serialization.Deserializer;
+
+import java.util.Map;
+
+public class JacksonDeserializer<T> implements Deserializer<T> {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private Class<T> tClass;
+
+    public JacksonDeserializer(Class<T> tClass) {
+        this.tClass = tClass;
+    }
+
+    public JacksonDeserializer() {
+
+    }
```

Comment:
I believe these can be removed since Kafka instantiates via reflection based on the default zero argument constuctor

Reply:
Removed